packages/shared/src/components/sidebar/ContributeSection.tsx

```diff
@@ -0,0 +1,86 @@
+import dynamic from 'next/dynamic';
+import React, { ReactElement, useContext, useState } from 'react';
+import AnalyticsContext from '../../contexts/AnalyticsContext';
+import FeaturesContext from '../../contexts/FeaturesContext';
+import EmbedIcon from '../icons/Embed';
+import LinkIcon from '../icons/Link';
+import { ListIcon, SidebarMenuItem } from './common';
+import { Section, SectionCommonProps } from './Section';
+
+const SubmitArticleModal = dynamic(
+  () =>
+    import(
+      /* webpackChunkName: "SubmitArticleModal" */ '../modals/SubmitArticleModal'
```

Comment:
Suggested camelCase:
```suggestion
      /* webpackChunkName: "submitArticleModal" */ '../modals/SubmitArticleModal'
```